packages/dashboard/src/CdcDashboardComponent.tsx

```diff
@@ -167,6 +175,7 @@ export default function CdcDashboard({ initialState, isEditor = false, isDebug =
                 data = transform.autoStandardize(data)
                 data = transform.developerStandardize(data, dataset.dataDescription)
               } catch (e) {
+                dispatchErrorMessages({ type: 'ADD_ERROR_MESSAGE', payload: e })
```

Comment:
you don't want to show the actual error message to the user. do:

console.error(e)

then dispatch the generic message to the user, ask UX if you need more guidance on what the message should be.

Reply:
Also you're missing a catch on the promise chain itself. fetch().then().catch( /* HANDLE ERROR */)

Reply:
I am not even a 100% sure what the error would be. I am going to put the same one as the other 'There was a problem returning data. Please try again.' for the user.